content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -29,6 +29,8 @@ summary: |
   unpacking the reasons to choose
   one platform over another.
   Let's start with responsive web apps.
+card: large
```

Comment:
Added this partially to test the automatic conversion of `large` cards (in the Posts list) down to `feature` cards (on the homepage). Feel free to change this if we don't want this post large or featured. (It looks like we're not using `card: feature` very frequently, at least recently?)

Reply:
@jgerigmeyer ah, ok, this confirms my question from above. I wonder if we need a `card: feature` in the long run. I feel like having the three levels of posts is a little distracting but that is definitely not a part of this story.

Reply:
Sure -- I'd defer to you and @SondraE and @mirisuzanne on that, but agreed it's not part of this story.